contracts/test-hs/README.md

```diff
@@ -0,0 +1,13 @@
+# Cleveland tests on minter-sdk contracts
+
+This folder contains:
+
+* A Haskell library with the bindings for the contracts' data types.
+* A test project with tests suites for the contracts.
+
+## Build and test instructions
+
+You need [Stack](http://haskellstack.org/) to build this package.
+
+See Makefile for how to run tests, `make test` should be enough for simple runs.
+Compiled `.tz` contract must already be in the `bin` folder.
```

Comment:
```suggestion
The compiled `fa2_swap.tz` contract must already be in the `bin` folder.
```
I'd mention the file name as well given that the tests depend on it having that specific name.